spec/services/discovery_report_spec.rb

```diff
@@ -102,13 +102,12 @@
     end
 
     it 'process_dobj gives expected output for one dobj' do
-      allow(dobj).to receive(:druid).and_return(DruidTools::Druid.new('kk203bw3276'))
       expect(report.send(:process_dobj, dobj).as_json).to eq(
-        druid: 'druid:kk203bw3276',
-        errors: { dupes: true },
+        druid: 'druid:jy812bp9403',
+        errors: {},
         counts: {
-          total_size: 254_802,
-          mimetypes: { 'image/tiff' => 4, 'image/jp2' => 2 },
+          total_size: 127_401,
+          mimetypes: { 'image/tiff' => 2, 'image/jp2' => 1 },
           filename_no_extension: 0
```

Comment:
our previous discovery was incorrectly counting files twice and our expectations were just adjusted to hide this bug ... we now stage files correctly, and these expectations match reality now